### DISCUSSION TITLE
This project should look for more maintainers

### DISCUSSION THREAD

**Discussion by samchouse** [shadcn-ui/ui#1374] [General]
I just wanted to start off by thanking you, shadcn, for this awesome UI library that you've created, it's helped a lot of us make amazing web UI's with little effort.

With that said, this project has grown very rapidly and looking at the commit history, open issues and open PRs, those seem to be falling behind. I understand that you're the only maintainer and you have things that take priority over this, which is fine, and your work and effort is greatly appreciated. I would imagine that being the sole maintainer of a project with nearly 30k stars is incredibly hard and tiring. In this case, I think that it would help you and this project if you're able to get some more maintainers onboard. This way the project can continue to advance even when you're unavailable.

**Comment by SoyDiego** [comment]
+1 
I did PRs and I understand he cannot check all the things. I hope more mantainers can help here :)

**Comment by Andrew-Chen-Wang** [comment]
How about making a separate repo given the lack of additional maintainers and slow merging?
